### ISSUE TITLE
chore(deps): update v8 to Angular 19 (major) 

### ISSUE THREAD

**Issue by gfellerph** [swisspost/design-system#4157]
- `@angular-devkit/build-angular`,
- `@angular-eslint/builder`,
- `@angular-eslint/eslint-plugin`,
- `@angular-eslint/eslint-plugin-template`,
- `@angular-eslint/template-parser`,
- `@angular/animations`,
- `@angular/cli`,
- `@angular/common`,
- `@angular/compiler`,
- `@angular/compiler-cli`,
- `@angular/core`,
- `@angular/forms`,
- `@angular/localize`,
- `@angular/platform-browser`,
- `@angular/platform-browser-dynamic`,
- `@angular/router`

**Comment by gfellerph** [comment]
On hold until cargo is live

**Comment by alizedebray** [comment]
Blocked until #4566 is merged,

**Comment by MonsieurBon** [comment]
When can we expect the update to angular 19 for v8?

**Comment by manuel-rw** [comment]
We have quite a few dependency updates, that are being blocked by the dependency conflict of the Design System.
As far as I understand, the Prime NG package is separate and independent - so would it be possible to add support for Angular 19 in the main packages and lift the Prime NG package when capacity is available, since most projects do not use it?

**Comment by MonsieurBon** [comment]
@gfellerph @alizedebray What is the status of this issue?

**Comment by gfellerph** [comment]
@alizedebray is working on it as a top priority. It's technically not the easiest feat due to internal repo structure hence the delay. We're sorry to keep you waiting and are looking forward to loosen the tight dependency on a specific Angular version in the future.
